euphoria-spark/src/main/java/cz/seznam/euphoria/spark/UnaryFunctorWrapper.java

```diff
@@ -26,31 +28,47 @@
 class UnaryFunctorWrapper<WID extends Window, IN, OUT>
         implements FlatMapFunction<SparkElement<WID, IN>, SparkElement<WID, OUT>> {
 
-  private final FunctionCollectorMem<OUT> context;
   private final UnaryFunctor<IN, OUT> functor;
+  private final AccumulatorProvider.Factory accumulatorFactory;
+  private final Settings settings;
 
-  public UnaryFunctorWrapper(UnaryFunctor<IN, OUT> functor) {
+  private FunctionCollectorMem<OUT> cachedCollector;
```

Comment:
Yes, you're right. Theoretically this should be `transient`. It worked only because the collector is initially `null`. I'll fix it.